robyn/__main__.py

```diff
@@ -30,44 +44,48 @@ def create_robyn_app():
     # Create the main application file
     app_file_path = os.path.join(project_dir, "app.py")
     with open(app_file_path, "w") as f:
-        f.write(
-            """
-from robyn import Robyn
-
-app = Robyn(__file__)
-
-@app.get("/")
-def index():
-    return "Hello World!"
-
-
-if __name__ == "__main__":
-    app.start()
-
-            """
-        )
+        if project_type == "sqlite":
+            f.write(open("robyn/scaffold/sqlite/app.py", "r").read())
+        elif project_type == "postgres":
+            f.write(open("robyn/scaffold/postgres/app.py", "r").read())
+            # copy README.md
+            readme_path = os.path.join(project_dir, "README.md")
+            with open(readme_path, "w") as f_r:
+                f_r.write(open("robyn/scaffold/postgres/README.md", "r").read())
+        elif project_type == "mongo":
+            f.write(open("robyn/scaffold/mongo/app.py", "r").read())
+        elif project_type == "sqlalchemy":
+            f.write(open("robyn/scaffold/sqlalchemy/app.py", "r").read())
+        elif project_type == "prisma":
+            f.write(open("robyn/scaffold/prisma/app.py", "r").read())
+            # copy schema.prisma
+            schema_path = os.path.join(project_dir, "schema.prisma")
+            with open(schema_path, "w") as f_s:
+                f_s.write(open("robyn/scaffold/prisma/schema.prisma", "r").read())
+            # copy README.md
+            readme_path = os.path.join(project_dir, "README.md")
+            with open(readme_path, "w") as f_r:
+                f_r.write(open("robyn/scaffold/prisma/README.md", "r").read())
+        else:
+            f.write(open("robyn/scaffold/no-db/app.py", "r").read())
```

Comment:
I wonder if there is a cleaner way to implement this?

can you store the file_name and file_paths in a dictionary and then try accessing it?

@IdoKendo do you have any thoughts on this?

Reply:
I would simplify this whole block to something like these 3 lines:
```py
base_path = f"robyn/scaffold/{project_type}"
shutil.copyfile(f"{base_path}/README.md", readme_path)
shutil.copyfile(f"{base_path}/app.py",  app_file_path)
```
Just make sure the `project_type` default is `no-db` 😃 
Can add some guard clauses to be extra safe, but I think `InquirerPy` does that for you out-of-the-box.

Reply:
Upon further review of the code, can even do it as simple as:
```py
from distutils.dir_util import copy_tree

copy_tree(project_dir, f"robyn/scaffold/{project_type}")
```

Reply:
This is much simpler but at the time I didn't want to add a new dependency. I've switched to this now.